compressor/storage.py

```diff
@@ -85,6 +86,37 @@ def save(self, filename, content):
         return filename
 
 
+class BrotliCompressorFileStorage(CompressorFileStorage):
+    """
+    The standard compressor file system storage that gzips storage files
+    additionally to the usual files.
+    """
+    chunk_size = 1024
```

Comment:
Maybe someone will be using it for compressing video files or something big. Anyway you do not lose anything using this code.